src/EditorFeatures/CSharpTest2/Recommendations/AwaitKeywordRecommenderTests.cs

```diff
@@ -1,120 +0,0 @@
-        [CombinatorialData]
-        public async Task TestStatementInMethod(bool isAsync, bool topLevelStatement)
-        {
-            await VerifyKeywordAsync(AddInsideMethod(
-@"$$", isAsync: isAsync, topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
-        }
-
-        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        [CombinatorialData]
-        public async Task TestExpressionInAsyncMethod(bool topLevelStatement)
-        {
-            await VerifyKeywordAsync(AddInsideMethod(
-@"var z = $$", isAsync: true, topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
-        }
-
-        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        [CombinatorialData]
-        public async Task TestUsingStatement(bool topLevelStatement)
-        {
-            await VerifyAbsenceAsync(AddInsideMethod(
-@"using $$", topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestUsingDirective()
-            => await VerifyAbsenceAsync("using $$");
-
-        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestGlobalUsingDirective()
-            => await VerifyAbsenceAsync("global using $$");
-
-        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        [CombinatorialData]
-        public async Task TestForeachStatement(bool topLevelStatement)
-        {
-            await VerifyAbsenceAsync(AddInsideMethod(
-@"foreach $$", topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
-        }
-
-        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        [CombinatorialData]
-        public async Task TestNotInQuery(bool topLevelStatement)
-        {
-            await VerifyAbsenceAsync(AddInsideMethod(
-@"var z = from a in ""char""
-          select $$", isAsync: true, topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
-        }
-
-        [WorkItem(907052, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/907052")]
-        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        [CombinatorialData]
-        public async Task TestInFinally(bool topLevelStatement)
-        {
-            await VerifyKeywordAsync(AddInsideMethod(
-@"try { }
-finally { $$ }", isAsync: true, topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
-        }
-
-        [WorkItem(907052, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/907052")]
-        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        [CombinatorialData]
-        public async Task TestInCatch(bool topLevelStatement)
-        {
-            await VerifyKeywordAsync(AddInsideMethod(
-@"try { }
-catch { $$ }", isAsync: true, topLevelStatement: topLevelStatement), options: CSharp9ParseOptions);
```

Comment:
did we effectively move all these tests over?

Reply:
Yep. They are now `AwaitCompletionProviderTests`